ui/ui-config/pages/index.tsx

```diff
@@ -18,12 +19,12 @@ export default function Page(props) {
 
   useEffect(() => {
     if (navigationMode === "config:authenticated") {
-      getSetupSteps(props);
+      checkSetupSteps(props);
```

Comment:
Changed this for clarity.  It doesn't return the setupSteps so it didn't make sense to me to call it a "getter".  It just checks if they're done and decides what CTA to show.